data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/web/DataControllerTest.java

```diff
@@ -49,64 +48,70 @@
 /**
  * Tests for the DataController web service endpoint.
  */
-@WebMvcTest(controllers = {DataController.class})
-@ContextConfiguration(classes = {DataControllerTest.class, DataController.class})
+@WebMvcTest(DataController.class)
+@ContextConfiguration(classes = {DataController.class})
 class DataControllerTest {
-    private static final ObjectMapper MAPPER = new ObjectMapper();
 
     @Autowired
-    private DataController controller;
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper mapper;
+
     @MockBean
     private AuditableDataService serviceMock;
+
     @MockBean
     private AuditMessageService auditMessageServiceMock;
-    @Autowired
-    private MockMvc mockMvc;
 
     @Test
     void testContextLoads() {
-        assertThat(controller).isNotNull();
         assertThat(mockMvc).isNotNull();
+        assertThat(serviceMock).isNotNull();
+        assertThat(auditMessageServiceMock).isNotNull();
```

Comment:
Can we add in some `as()` descriptions for these assertions.

Reply:
done, but there is not much to these.  They just tell me that I have the auto-wiring working